src/smokeTest/java/uk/gov/hmcts/reform/ccd/document/am/SmokeTest.java

```diff
@@ -0,0 +1,148 @@
+package uk.gov.hmcts.reform.ccd.document.am;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import net.serenitybdd.junit.spring.integration.SpringIntegrationSerenityRunner;
+import net.serenitybdd.rest.SerenityRest;
+import org.hamcrest.Matchers;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import uk.gov.hmcts.reform.ccd.document.am.utils.IdamUtils;
+
+import static org.springframework.http.HttpHeaders.CONTENT_TYPE;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+import static uk.gov.hmcts.reform.ccd.document.am.apihelper.Constants.AUTHORIZATION;
+import static uk.gov.hmcts.reform.ccd.document.am.apihelper.Constants.BEARER;
+import static uk.gov.hmcts.reform.ccd.document.am.apihelper.Constants.SERVICE_AUTHORIZATION2;
+
+@SpringBootTest
+@RunWith(SpringIntegrationSerenityRunner.class)
+//@ConfigurationProperties()
+public class SmokeTest extends BaseTest {
```

Comment:
No need for smoke tests written in Serenity Rest…
A few of functional tests can do the same. See data store develop branch

Reply:
needs actioning @nitishsingla1234 , @aashishaggarwal123

Reply:
Removed this class and tagged a F-000 functional test as smoke test.